source/code/go/src/plugins/out_oms.go

```diff
@@ -50,6 +48,8 @@ func FLBPluginFlush(data unsafe.Pointer, length C.int, tag *C.char) int {
 
 // FLBPluginExit exits the plugin
 func FLBPluginExit() int {
+	KubeSystemContainersRefreshTicker.Stop()
```

Comment:
Is it possible to check if its started before stopping ?

Reply:
I believe the ticker automatically starts by itself. So the check is not required.